.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         if: success()
         uses: codecov/codecov-action@v1
         with:
-          file: ./coverage.xml
+          file: ./coverage_core.xml,./coverage_mosaic.xml,./coverage_application.xml,
```

Comment:
I haven't used codecov before, but this might give you three different reports on the dashboard, instead of a single report with three different modules.